### ISSUE TITLE
Create build/deploy Documentation

### ISSUE THREAD

**Issue by phildree** [amosproj/amos2023ss01-apache-pulsar-ui#87]
## User story
1. As a user
2. I need documentation
3. So that I know how to use this project

## Acceptance criteria
* [ ] Create documentation on how to use your software (user documentation)
* [ ] Create documentation on the design of your software (architecture, code) (design documentation)
* [ ] Create documentation on how to build and deploy your project (build/deploy documentation)

=> Documentation should be added to the [Wiki](https://github.com/amosproj/amos2023ss01-apache-pulsar-ui/wiki)
=> Wiki Pages should be uploaded to the Deliverables folder: _sprint-06/{user|design|build}-documentation.pdf_

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by ahaverkamp** [comment]
Coudl you change the title to just documentation, because we need a user docu and a design docu as well as the build/deploy docu : )
